Update veeam-backup-and-replication.md

adrianwells opened the pull request:
Adding 12.1 release information. EoL date appears to be same as 12.0 per https://www.veeam.com/product-lifecycle.html

adrianwells committed:
Update veeam-backup-and-replication.md

Adding 12.1 release information. EoL date appears to be same as 12.0 per https://www.veeam.com/product-lifecycle.html

marcwrobel reviewed (commented):
Not sure this must be documented as 12.1 is part of the 12 lifecycle.

On https://www.veeam.com/kb4420, you can read "This update has been superseded by [Veeam Backup & Replication 12.1](https://www.veeam.com/kb2680?ad=in-text-link).". This is similar to release 11 where you can see "This update has been superseded by [Veeam Backup & Replication 11a](https://www.veeam.com/kb4215?ad=in-text-link).".

Considering we did not document 11a, I think we shouldn't document 12.1. WDYT ?

adrianwells committed:
Update veeam-backup-and-replication.md

Added 11a release to be consistent with adding 12.1 release.

adrianwells committed:
Update veeam-backup-and-replication.md

Updated based on https://github.com/endoflife-date/endoflife.date/pull/4418#issuecomment-1891055314
